com.unity.multiplayer.mlapi/Runtime/Configuration/MLAPIConstants.cs

```diff
@@ -64,8 +60,8 @@ internal static class MLAPIConstants
             "",
             "",
             "",
-            "",
-            "",
+            "MLAPI_STD_SERVER_RPC",
```

Comment:
I guess we wanted for sure these to be 30 & 31

Reply:
yeah, these string indices are matching with message type IDs (const integers). I think I can move them up rather than putting them to the bottom since old RPC message IDs created a space for it — I will do that.